src/app/users/users.component.html

```diff
@@ -1 +1,20 @@
-<p>users works!</p>
+<br />
+<div *ngFor="let u of users" class="card" style="width: 18rem">
+  <img
+    src="https://upload.wikimedia.org/wikipedia/commons/d/d3/User_Circle.png"
+    class="card-img-top"
+    alt="..."
+    style="max-height: 40px; max-width: 40px"
+  />
+  <div class="card-body">
+    <h5 class="card-title">{{ u.name }}</h5>
+  </div>
+  <ul class="list-group list-group-flush">
+    <li class="list-group-item">{{ u.email }}</li>
+    <li class="list-group-item">{{ u.password }}</li>
+  </ul>
+  <div class="card-body">
+    <a class="card-link">Card link</a>
```

Comment:
why you  ship trash to the markup? I mean, things that you are not going to use.

If you want, you can mock something  eg: 
instead of
<a class="card-link">Card link</a>

do something useful or with a meaning , like: 

<a class="card-link">Save</a>

The save button

Reply:
I changed both links to "Edit user" and "Delete User" thanks

Reply:
Great,done!